# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arpwatch/hotfix-time.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- arpwatch-3.8/arpwatch.h.vanilla	2025-03-08 10:11:52.978271918 +0100
+++ arpwatch-3.8/arpwatch.h	2025-03-08 10:12:06.871114505 +0100
@@ -34,3 +34,5 @@
 extern int suppress;
 extern const char *watcher;
 extern const char *watchee;
+
+#include <time.h>
